Specification-Draft.MD

```diff
@@ -27,30 +55,84 @@ Examples of End Points include:
-- Chain:  in this case the workflow is completely handed off from one app to another (similar to linking).   Currently, this is the primary focus in FDC3
-- Client-Service: In these cases,  a Client invokes a Service via the Intent, the Service performs some function, then passes the workflow back to the Client.  Usually, there is a data payload associated with this workflow and a request/response pattern.
+## 2.2 Raising Intents
+Often, we want to link from one app to another to dynamically create a workflow.  Enabling this without requiring prior knowledge between apps is a key goal of FDC3.
+
+Intents provide a way for an app to request functionality from another app and defer the discovery and launching of the destination app to the Desktop Agent.  There are multiple models for interop that Intents can support.
+
+- **Chain**:  In this case the workflow is completely handed off from one app to another (similar to linking).   Currently, this is the primary focus in FDC3
+- **Client-Service**: A Client invokes a Service via the Intent, the Service performs some function, then passes the workflow back to the Client.  Typically, there is a data payload type associated with this intent that is published as the standard contract for the intent.
+- **Remote API**: An app wants to remote an entire API that it owns to another App.  In this case, the API for the App cannot be standardized.  However, the FDC3 API can address how an App connects to another App in order to get access to a proprietary API.
+
+### 2.2.1 Intent Resolution
+Raising an Intent will return a Promise-type object that will resolve/reject based on a number of factors.
+
+#### 2.2.1.1 Resolve
+- Intent was resolved unambigiously and the recieving app was launched successfully.
+- Intent was ambigious, a resolution was chosen by the end user and the chosen application was launched succesfully.
+
+#### 2.2.1.2 Reject
+- An app matching the intent was not found.
+- A match was found, but the recieving app failed to launch.
+- The intent was ambiguous and the resolver experienced an error.
+
+#### 2.2.1.3 Resolution Object
+If the raising of the intent resolves (or rejects), a standard object will be passed into the resolver function with the following format:
+
+```javascript
+{
+    source: String,
+    data: Object, (optional)
+    version: String, 
+    error: Object (optional)
+}
+```
+- *source* = identifier for the Application resolving the intent (null if the intent could not be resolved)
+- *data* = return data structure - if one is provided for the given intent
+- *version* = the version number of the Intents schema being used
+- *error* = errror object (name, message), if the intent rejects 
+**Note:** The *success* property is to indicate 
+
+For example
+
+```javascript
+agent.raiseIntent('StageOrder').then(
```

Comment:
this code sample doesn't look right, you've got a `Promise.then` function that has two `result` arguments. It should be a single value - did you intend to have the second function as a `catch`?